packages/website/components/Topbar.tsx

```diff
@@ -60,84 +63,75 @@ const Logo = () => (
     viewBox="0 0 90 90"
     enableBackground="new 0 0 90 90"
   >
-    <circle fill="#ffffff" cx="45" cy="10" r="10" />
-    <circle fill="#ffffff" cx="10" cy="10" r="10" />
-    <circle fill="#ffffff" cx="80" cy="10" r="10" />
-    <circle fill="#ffffff" cx="10" cy="45" r="10" />
-    <circle fill="#ffffff" cx="45" cy="45" r="10" />
-    <circle fill="#ffffff" cx="10" cy="80" r="10" />
+    <circle fill={tokens.blue700} cx="45" cy="10" r="10" />
+    <circle fill={tokens.blue700} cx="10" cy="10" r="10" />
+    <circle fill={tokens.blue700} cx="80" cy="10" r="10" />
+    <circle fill={tokens.blue700} cx="10" cy="45" r="10" />
+    <circle fill={tokens.blue700} cx="45" cy="45" r="10" />
+    <circle fill={tokens.blue700} cx="10" cy="80" r="10" />
```

Comment:
Did you try to pass fill to the `svg`?

I think it's possible and then we don't need to repeat the color several times

Reply:
yeah, probably. I will handle it differently.